renovate.json

```diff
@@ -1,46 +1,87 @@
+
 {
- "extends": ["config:base"],
- "baseBranches": ["master"],
- "bumpVersion": null,
- "packageRules": [
-   {
-     "depTypeList": ["devDependencies"],
-     "updateTypes": ["minor"],
-     "automerge": false
-   }
- ],
- "prHourlyLimit": 5,
- "prConcurrentLimit": 5,
- "rangeStrategy": "bump",
- "renovateFork": true,
- "includeForks": true,
- "regexManagers": [
+  "extends":[
+    "config:base"
+  ],
+  "enabled": "true",
+  "baseBranches": ["master"],
+  "timezone": "Europe/Helsinki",
+  "packageRules": [
     {
-      "fileMatch": ["^lib/common.sh$"],
-      "matchStrings": ["MINIKUBE_VERSION:-\"(?<currentValue>.*?)\"}"],
-      "depNameTemplate": "kubernetes/minikube",
-      "datasourceTemplate": "github-releases"
+      "depTypeList": ["devDependencies"],
+      "matchUpdateTypes": ["minor"],
+      "automerge": true
```

Comment:
I see this was set to `false` before? Not sure if it matters since we anyway have prow that manages the PRs but got curious why it changed.

Reply:
When it is set to false, Renovate only creating a branch and it is not creating a PR. So, to create PR automatically I have set it to "true"